generators/deployment/index.js

```diff
@@ -33,16 +34,23 @@ module.exports = class extends Generator {
 		this.toolchainConfig = {};
 		this.pipelineConfig = {
 			buildJobProps : {artifact_dir: "''"},
-			services: this.bluemix.services
+			services: this.bluemix.services,
+			deployment: {type: 'CF', name: this.bluemix.name}
 		};
-		this.name = undefined
+
+		this.name = undefined;
 		if(this.bluemix.server) {
 			this.name = this.bluemix.server.name;
 			this.manifestConfig = Object.assign(this.manifestConfig, this.bluemix.server);
+			this.pipelineConfig.deployment = Object.assign(this.pipelineConfig.deployment, this.bluemix.server.cloudDeploymentOptions);
+			this.pipelineConfig.deployment.type = this.bluemix.server.cloudDeploymentType || 'CF';
```

Comment:
The variable name this.pipelineConfig.deployment seems deceptive as the content is not only used in the pipeline file, but also the deploy file, maybe this.deployment would be better